scanner/updater/export.go

```diff
@@ -62,18 +62,23 @@ func Export(ctx context.Context, outputDir string) error {
 	outOfTree = append(outOfTree, updaterSet.Updaters())
```

Comment:
Nit: Maybe simpler:

```suggestion
	outOfTree := append(make([][]driver.Updater, 1), updaterSet.Updaters())
```

And you don't need:

```
	outOfTree := [][]driver.Updater{
		make([]driver.Updater, 0),
	}
```